apps/bankingAPI_web/test/bankingAPI_web/controllers/user_controller_test.exs

```diff
@@ -1,10 +1,36 @@
 defmodule BankingAPIWeb.UserControllerTest do
   use BankingAPIWeb.ConnCase, async: true
 
-  alias BankingAPI.Users.Schemas.User
   alias BankingAPI.Repo
+  alias BankingAPI.Users.Schemas.User
 
   describe "POST /api/users" do
+    test "sucess with 200 and user created", ctx do
+      name = "Renan infinity PR"
+      email = "infinityPR@mail.com"
+
+      input = %{
+        "name" => name,
+        "email" => email,
+        "email_confirmation" => email
+      }
+
+      assert Enum.empty?(Repo.all(User))
```

Comment:
Não precisa disso, pode partir da premissa de que não vai ter um user inserido no início do teste